packages/libs/src/services/solana/transactionHandler.ts

```diff
@@ -286,16 +286,18 @@ export class TransactionHandler {
     } catch (e) {
       // Rarely, this intiial send will fail
       logger.error(`transactionHandler: Initial send failed: ${e}`)
-      let errorCode = null
-      let error = null
-      if (e instanceof Error) {
-        error = e.message
-        errorCode = this._parseSolanaErrorCode(error)
-      }
-      return {
-        res: null,
-        error,
-        errorCode
+      if (!retry) {
```

Comment:
I like this, but maybe we should just let it fall through to the other error handling